CHANGELOG.md

```diff
@@ -5,72 +5,40 @@ All notable changes to this project will be documented in this file.
-- Load all the values along with their key names into the field update method [(#17973)](https://github.com/ManageIQ/manageiq/pull/17973)
-- Fix Physical Disks save inventory [(#17972)](https://github.com/ManageIQ/manageiq/pull/17972)
-- Fix Canisters save inventory [(#17966)](https://github.com/ManageIQ/manageiq/pull/17966)
-- Add Switch model to RBAC [(#17964)](https://github.com/ManageIQ/manageiq/pull/17964)
-- Consolidate production env checks [(#17957)](https://github.com/ManageIQ/manageiq/pull/17957)
-- Add replication set-up methods to be queued on UI side [(#17956)](https://github.com/ManageIQ/manageiq/pull/17956)
-- Adds task to subservices for correct retirement completion [(#17912)](https://github.com/ManageIQ/manageiq/pull/17912)
-- Enable cancel operation for service template transformation plan request [(#17825)](https://github.com/ManageIQ/manageiq/pull/17825)
-- Add Ownership and Tenancy Mixins to Authentication [(#17731)](https://github.com/ManageIQ/manageiq/pull/17731)
-
-## Unreleased as of Sprint 94 ending 2018-09-10
+- Don't need the name since it's mixed in... [(#18117)](https://github.com/ManageIQ/manageiq/pull/18117)
+Check if class is taggable before attempting to process tag expression [(#18114)](https://github.com/ManageIQ/manageiq/pull/18114)
+Add object retirement_requester [(#18113)](https://github.com/ManageIQ/manageiq/pull/18113)
+RestClient: Support percent encoded proxy user/pass [(#18105)](https://github.com/ManageIQ/manageiq/pull/18105)
+Don't use special characters in ansible passwords [(#18092)](https://github.com/ManageIQ/manageiq/pull/18092)
+Disable transactions in locking examples [(#18089)](https://github.com/ManageIQ/manageiq/pull/18089)
+Parse automation attrs correctly [(#18084)](https://github.com/ManageIQ/manageiq/pull/18084)
+Rails 5.0/5.1 compatibility: Define through association before has many through [(#18080)](https://github.com/ManageIQ/manageiq/pull/18080)
+Rails 5.0/5.1 Use o.reload.assoc assoc(true) is gone [(#18079)](https://github.com/ManageIQ/manageiq/pull/18079)
+Rails 5.0/5.1 prepare_binds_for_database is gone [(#18078)](https://github.com/ManageIQ/manageiq/pull/18078)
+Rails 5.1/5.0 compatibility: Use constant since string/symbol was removed [(#18077)](https://github.com/ManageIQ/manageiq/pull/18077)
+Handle a blank value for the http_proxy host [(#18073)](https://github.com/ManageIQ/manageiq/pull/18073)
+Add flag for init of defaults in fields [(#18061)](https://github.com/ManageIQ/manageiq/pull/18061)
+Order custom buttons by array of ids [(#18060)](https://github.com/ManageIQ/manageiq/pull/18060)
+Add retirement initiator context [(#17951)](https://github.com/ManageIQ/manageiq/pull/17951)
+Add tenant filtering for templates in provisioning and summary pages [(#17851)](https://github.com/ManageIQ/manageiq/pull/17851)
+
+## Hammer Beta-1 - Released 2018-10-12
```

Comment:
Was Hammer cut off of master or hammer branch?

Reply:
Cutoff master. (however, we list upstream releases in both master and the branch)